jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -268,6 +277,8 @@ MethodHandle cloneWithNewType(MethodType newType) {
 		enforceArityLimit(original.kind, newType);
 		this.thunks = original.thunks;
 		this.previousAsType = original.previousAsType;
+		/* Touch thunks.invokeExactThunk so that its constant pool entry is resolved by the time it is used by the JIT */
+		long i = thunks.invokeExactThunk;
```

Comment:
This change isn't necessary as the only way to run this ctor is to already have a MH in hand.  The change to the other ctor should be sufficient.

Reply:
This change has been removed.